adafruit_wiznet5k/adafruit_wiznet5k_ntp.py

```diff
@@ -0,0 +1,62 @@
+# SPDX-FileCopyrightText: 2019 Brent Rubell for Adafruit Industries
```

Comment:
Author on L11 doesn't match the SPDX author
>  * Author(s): irinakim

Reply:
Author(s): Brent Rubell, irinakim
I'll revise the Author part.

Reply:
Unlike the existing adafruit_NTP, the added NTP library is favorable to the WIZNET5k library.

Reply:
Is there a way we could modify the existing CircuitPython NTP library for use with this library instead? I'd prefer that if possible, have one library consistently used across the network interfaces.

Reply:
I've looked up many ways, but I think it's going to hard to combine them.

Reply:
Understood, OK!